scripts/ci/build_and_push_image_sonar.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+
+# shellcheck disable=SC1091
+. venv/bin/activate
+echo "${quay_password:?}" | docker login "-u=${quay_user_name:?}" quay.io --password-stdin
```

Comment:
Depending on execution parameters of this shell script, this password could be printed to stdout by mistake.

Can you add an explicit `set +x` to this script, right before `docker login` and add a comment that it should not be removed, because it could cause the quay_password contents to be printed.

Optimally we would use a credentials helper, but none seems to exist for Quay.